neonvm/pkg/ipam/allocate.go

```diff
@@ -0,0 +1,60 @@
+package ipam
+
+import (
+	"context"
+	"fmt"
+	"net"
+
+	whereaboutsallocate "github.com/k8snetworkplumbingwg/whereabouts/pkg/allocate"
+	whereaboutslogging "github.com/k8snetworkplumbingwg/whereabouts/pkg/logging"
+	whereaboutstypes "github.com/k8snetworkplumbingwg/whereabouts/pkg/types"
+)
+
+func doAcquire(ctx context.Context, ipRange RangeConfiguration, reservation []whereaboutstypes.IPReservation, vmName string, vmNamespace string) (net.IPNet, []whereaboutstypes.IPReservation, error) {
```

Comment:
This line is quite long, can we break it up? There's some similarly long lines below as well.

Reply:
done